conn_test.go

```diff
@@ -95,6 +110,81 @@ func TestValidCommunicationWithAddressConfig(t *testing.T) {
 	check(aServer.Close(), t)
 }
 
+func TestValidCommunicationWithLoopbackAddressConfig(t *testing.T) {
```

Comment:
I should have added this test originally since I was intended on increasing the chances we get loopback addresses for when I was working on some offline code.